src/wallet/rpcwallet.cpp

```diff
@@ -3054,7 +3055,7 @@ UniValue listunspent(const JSONRPCRequest& request)
                 "      \"address\"   (string) pivx address\n"
                 "      ,...\n"
                 "    ]\n"
-                "4. watchonlyconfig  (numeric, optional, default=1) 1 = list regular unspent transactions, 2 = list only watchonly transactions,  3 = list all unspent transactions (including watchonly)\n"
+                "4. watchonlyconfig  (numeric, optional, default=1) 1 = list regular unspent transactions,  2 = list all unspent transactions (including watchonly)\n"
```

Comment:
this would be maybe better as a boolean argument now:
```cpp
"4. fAllowWatchOnly  (boolean, optional, default=false) include unspent outputs to watchonly addresses\n"

...

coinControl.fAllowWatchOnly = request.params.size() > 3 && request.params[2].get_bool();
```

Reply:
Yeah, can do it.
But.. I'm more thinking on remove the argument. It makes little sense to be only filtering watch-only utxo with this param and not other type of utxos.
Would go more generic and move the filtering responsibility to the RPC command caller. With #1755, `listunspent` caller will know whether a utxo is solvable/spendable or not pretty easily.

Reply:
Yeah. Could probably be out right removed.

Reply:
will work on it on a subsequent PR.